src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -106,9 +115,8 @@ public SortedSet(IEnumerable<T> collection, IComparer<T> comparer)
                 }
                 return;
             }
-            
-            int count;
-            T[] elements = EnumerableHelpers.ToArray(collection, out count);
+
+            T[] elements = EnumerableHelpers.ToArray(collection, out int count);
```

Comment:
@ViktorHofer That's not what I intended to say, I appreciated your advice, I was just making an off-the-cuff remark about how I liked skipping the declaration in my personal projects.